packages/tables/resources/views/components/row.blade.php

```diff
@@ -10,7 +11,7 @@
         'dark:hover:bg-gray-500/10' => $recordUrl && config('tables.dark_mode'),
         'even:bg-gray-100' => $striped,
         'dark:even:bg-gray-900' => $striped && config('tables.dark_mode'),
-    ]) }}
+    ])->class($recordClasses) }}
```

Comment:
Let's merge this in with the above using `array_merge` (need to support PHP 8.0, else could have used the spread operator)?